misc/scripts/updateGHpages.sh

```diff
@@ -1,10 +1,21 @@
 #!/bin/bash
-
-# Following script copies the markdown files from vmware:master to gh-pages
-# Local copy of vDVS project is needed before running this script (git clone https://github.com/vmware/docker-volume-vsphere.git)
-# Script has to be run from the docker-volume-vsphere/
-# e.g. ./misc/scripts/updateGHpages.sh
-
+set -e
+: '
+Following script copies the markdown files from vmware:master to gh-pages
+Local copy of vDVS project is needed before running this script (git clone https://github.com/vmware/docker-volume-vsphere.git)
+Script has to be run from the docker-volume-vsphere/
+e.g. ./misc/scripts/updateGHpages.sh
```

Comment:
minor: `./misc/scripts/updateGHpages.sh` => `./misc/scripts/<script_name>` .. it will be outdated soon when script name got rename.